package.json

```diff
@@ -19,6 +19,7 @@
     "tree-sitter-python": "^0.11.3"
   },
   "devDependencies": {
-    "coffeelint": "^1.10.1"
+    "coffeelint": "^1.10.1",
+    "atom-grammar-test": "^0.6.4"
```

Comment:
I believe this needs to be in the dependencies section, or else tests won't run on atom/atom (https://github.com/atom/language-html/commit/624139dbaf0536913e1ba854bef519267b4aabe6).  I don't think the situation has changed since that commit.

Reply:
weird.  ok.  fixed.